src/main/java/net/datafaker/providers/base/Internet.java

```diff
@@ -91,7 +91,7 @@ public String domainName() {
     }
 
     public String domainWord() {
-        return FakerIDN.toASCII(SINGLE_QUOTE.matcher(faker.name().lastName().toLowerCase()).replaceAll(""));
+        return FakerIDN.toASCII(faker.name().lastName().toLowerCase().replace("'", ""));
```

Comment:
Shouldn't lowercase here at least use Locale.ROOT?

Reply:
here it's name generated base on locale set by user, so in case it iis e.g. TR it should be being lower cased to turkish lowercase. Thus i think here it is ok to have locale from faker's context

Reply:
Is it using fakers context when not passed though?

Reply:
yes, good catch, 
added locale from context
thanks